src/fides/api/ops/api/v1/endpoints/dataset_endpoints.py

```diff
@@ -172,10 +233,10 @@ def patch_datasets(
     Given a list of dataset elements, create or update corresponding Dataset objects
```

Comment:
I believe this endpoint `{{host}}/connection/{{connection_key}}/dataset` should be deprecated once the UI has been updated to use the new endpoint above. 

Added some functionality here to make it still usable.  If a raw dataset is passed in, I write it to both the DatasetConfig.dataset field and the ctl_dataset record.

Reply:
Should there be a follow up ticket keeping track of all of the soon to be removed/deprecated routes?

Reply:
Good question Andrew, here's the follow-up ticket, we can wait to deprecate until the UI has been pointed to use the new endpoints https://github.com/ethyca/fides/issues/2092